web/templates/presence_row.hbs

```diff
@@ -1,26 +1,43 @@
-<li data-user-id="{{user_id}}" class="user_sidebar_entry {{#if is_current_user}}user_sidebar_entry_me {{/if}} narrow-filter {{#if faded}} user-fade {{/if}}">
+<li data-user-id="{{user_id}}" class="user_sidebar_entry {{#if user_list_style.WITH_AVATAR}}with_avatar{{/if}} {{#if is_current_user}}user_sidebar_entry_me {{/if}} narrow-filter {{#if faded}} user-fade {{/if}}">
     <div class="selectable_sidebar_block">
-        <span class="{{user_circle_class}} user_circle"></span>
-        <a class="user-presence-link"
-          href="{{href}}"
-          data-user-id="{{user_id}}"
-          data-name="{{name}}">
+        {{#unless user_list_style.WITH_AVATAR}}<span class="{{user_circle_class}} user_circle"></span>{{/unless}}
+        <div class="user-presence-row" data-user-id="{{user_id}}" data-name="{{name}}">
             {{#if user_list_style.WITH_STATUS}}
-                <div class="user-name-and-status-wrapper">
+                <a class="user-presence-link" href="{{href}}">
+                    <div class="user-name-and-status-wrapper">
+                        <div class="user-name-and-status-emoji">
+                            {{> user_full_name}}
+                            {{> status_emoji status_emoji_info}}
+                        </div>
+                        <span class="status-text">{{status_text}}</span>
+                    </div>
+                </a>
+            {{else if user_list_style.WITH_AVATAR}}
+                <div class="user-profile-picture">
+                    <img src="{{profile_picture}}"/>
```

Comment:
We should consider using the `lazyload` flag on this setting, probably, since fetching a ton of images all at once can be slow, and letting it defer those that are offscreen is probably helpful.

You may also need to add some more complex styling here to handle the case that a given user's avatar isn't cached yet.

Reply:
Added `loading="lazy" `

Reply:
Can you explain what you mean by styling for avatars that aren't cached yet? Cached in the browser or cached on the `person` object?

Reply:
Cached in the browser -- basically the question is what happens if you're on a slow network and loading this with nothing in the browser cache. It might look fine with no special code; but we should probably check since that is a thing some users will see.

Reply:
This is what it looks like. It's not the nicest but it also seems not that bad? Especially alongside the rest of how the page is loading. I'm not sure what our standard is but let me know if you'd like me to add some extra styling!

<img width="967" alt="image" src="https://github.com/user-attachments/assets/89c2cd9e-cac5-428a-9673-53132c24c614">

Reply:
Can you post that in the design thread? The sidebar showing the "avatar not loaded" state will also be visible when clicking into a conversation containing users that the browser hasn't cached yet, not just during early app loading, which is why I think it deserves special attention.

Reply:
(Probably this is not a blocker for merging this PR, but it is probably a thing we should aim to get to a nice state before declaring the feature finished)

Reply:
https://chat.zulip.org/#narrow/channel/101-design/topic/userlist.20avatar.20slow.20load